.github/workflows/build_wheels_windows.yml

```diff
@@ -131,13 +145,36 @@ jobs:
           XPU_VERSION: '2025.0'
         run: |
           cmd //c .\\test-infra\\.github\\scripts\\install_xpu.bat
+      - name: Checkout Target Repository (${{ env.REPOSITORY }})
+        if: inputs.architecture == 'arm64'
+        uses: actions/checkout@v4
+        with:
+            repository: ${{ env.REPOSITORY }}
+            ref: ${{ env.REF }}
+            path: ${{ env.REPOSITORY }}
+            submodules: recursive
```

Comment:
Doesn't this workflow already do this? Why is windows arm64 so different here?

Just a general note: if we're making so many conditionals specifically for windows arm64 why not just make a completely separate workflow

Reply:
I noticed that the setup-binary-build action handles the checkout, but it also sets up conda environment, which we can't use for Windows on arm. That’s why I added a separate checkout step just for arm64 - to avoid the conda setup